back/app/models/permission.rb

```diff
@@ -24,7 +24,7 @@ class Permission < ApplicationRecord
   PERMITTED_BIES = %w[everyone everyone_confirmed_email users admins_moderators verified].freeze
   ACTIONS = {
     # NOTE: Order of actions in each array is used when using :order_by_action
-    nil => %w[visiting following posting_initiative commenting_initiative reacting_initiative attending_event],
```

Comment:
We will also need to delete existing permissions on existing platforms.

Reply:
Discussed with Sebi. This will be cleaned up by CI on deploy.